bindings/rust/build.rs

```diff
@@ -2,23 +2,21 @@ fn main() {
     let src_dir = std::path::Path::new("src");
 
     let mut c_config = cc::Build::new();
-    c_config.include(&src_dir);
-    c_config
-        .flag_if_supported("-Wno-unused-parameter")
-        .flag_if_supported("-Wno-unused-but-set-variable")
-        .flag_if_supported("-Wno-trigraphs");
+    c_config.std("c11").include(src_dir);
+
     #[cfg(target_env = "msvc")]
     c_config.flag("-utf-8");
 
     let parser_path = src_dir.join("parser.c");
     c_config.file(&parser_path);
+    println!("cargo:rerun-if-changed={}", parser_path.to_str().unwrap());
 
-    // If your language uses an external scanner written in C,
-    // then include this block of code:
+    // NOTE: if your language uses an external scanner, uncomment this block:
```

Comment:
I think we should do this. But maybe it would be also a separate PR

Reply:
I'll fix this in this branch.

(It also makes me curious about making the upgrades easier in `tree-sitter` itself, given that e.g. filesystem state should be easy to inspect during upgrades)